packages/android-client/FishjamClient/src/main/java/com/fishjamcloud/client/CommandsQueue.kt

```diff
@@ -1,28 +1,31 @@
 package com.fishjamcloud.client
 
-import kotlinx.coroutines.CancellationException
-import kotlinx.coroutines.Job
+import kotlin.coroutines.resume
+import kotlin.coroutines.suspendCoroutine
 
 internal class CommandsQueue {
   private var commandsQueue: ArrayDeque<Command> = ArrayDeque()
   var clientState: ClientState = ClientState.CREATED
 
-  fun addCommand(command: Command): Job {
+  suspend fun executeCommand(command: Command) {
```

Comment:
I do not like name execute command, because it does not always execute command here. I think that `addCommand` is a better name.